.bazelci/presubmit.yml

```diff
@@ -28,17 +67,55 @@ platforms:
     # Disabled e2e tests that pull from localhost in nested workspace
     - "-//testing/new_pusher_tests/..."
     # tests/contrib/test_compare_ids_test_* expect 'bazel' on path
+    # container_test and configs_test targets require 'docker' and are disabled
     test_flags:
     - "--action_env=PATH"
-    # For tests/container:set_env_make_vars
-    - "--define=ENV_KEY=my_key"
-    - "--define=ENV_VALUE=my_value"
     - "--test_output=errors"
   ubuntu1804:
     test_targets:
     - "--"
-    - "..."
+    - "//container/go/pkg/compat:go_default_test"
```

Comment:
Add doc here indicating these tests do not require docker

Reply:
Done